src/script/lua_api/l_storage.cpp

```diff
@@ -57,6 +57,11 @@ StorageRef::StorageRef(ModMetadata *object):
 {
 }
 
+StorageRef::~StorageRef()
+{
+	delete m_object;
```

Comment:
why StorageRef delete m_object ? it's passed by the constructor and seems to be owned by something else no ?

Reply:
`core.get_mod_storage` creates the pointer, which is then passed to the server or client which then constructs the StorageRef object with that pointer.

Reply:
yeah it's on the Lua master stack, i should reverify my code to ensure it's not stored elsewhere. if not we can remove it safely